cicecore/cicedynB/infrastructure/ice_grid.F90

```diff
@@ -155,7 +155,7 @@ module ice_grid
       real (kind=dbl_kind), dimension (:,:,:), allocatable, public :: &
          hm     , & ! land/boundary mask, thickness (T-cell)
          bm     , & ! task/block id
-         uvm    , & ! land/boundary mask, velocity (U-cell)
+         uvm    , & ! land/boundary mask, velocity (U-cell) - land if one point is one
```

Comment:
I'm not sure about that comment means exactly .
Also below in `alloc_grid` you used a different wording: "water in case of all water point"

Reply:
For concistency. Comment is removed